server.js

```diff
@@ -4,10 +4,33 @@ const path = require('path')
 const morgan = require('morgan')
 const bodyParser = require('body-parser')
 const pug = require('pug')
+const passport = require('passport')
+const GoogleStrategy = require('passport-google-oauth').Strategy
 const admin = require('./routes/admin.js')
 const routes = require('./routes/index.js')
+const mongoose = require('mongoose');
+const flash    = require('connect-flash');
+const cookieParser = require('cookie-parser');
+const session      = require('express-session')
+
+const configDB = require('./config/database.js');
+mongoose.createConnection(configDB.url);
+
+app.use(morgan('dev'));
+app.use(cookieParser());
+app.use(bodyParser());
+
+app.use(session({ secret: 'iloveshakashakashaka' }));
+app.use(passport.initialize());
+app.use(passport.session());
+require('./config/passport.js')(passport)
```

Comment:
This require feels out of place. Could it go up at the top with its other require friends so its not so lonely?